benchmarks/benchmark_tests.yaml

```diff
@@ -39,3 +39,17 @@
     timeout: 3600 # 1hr
     prepare: sleep 0
     script: python distributed/test_distributed.py
+
+# - name: test_1k_20k
```

Comment:
Is this supposed to be commented out?

Reply:
Yes, right now, our internal pipeline can't scale to 1k node in a short time.

Reply:
why don't we just set the longer timeout? It is better testing than skipping although it takes long time.